sdk/forms/src/main/java/com/klaviyo/forms/bridge/KlaviyoJsBridge.kt

```diff
@@ -11,20 +11,48 @@ import com.klaviyo.forms.webview.JavaScriptEvaluator
 internal class KlaviyoJsBridge : JsBridge {
     @Suppress("EnumEntryName", "ktlint:enum-entry-name-case")
     private enum class HelperFunction {
-        setProfile,
-        dispatchLifecycleEvent
+        profileMutation,
+        lifecycleEvent,
+        openForm,
+        closeForm
     }
 
-    override fun setProfile(profile: ImmutableProfile) = evaluateJavascript(
-        HelperFunction.setProfile,
+    override val handshake: List<HandshakeSpec> = listOf(
+        HandshakeSpec(
+            type = HelperFunction.profileMutation.name,
+            version = 1
+        ),
+        HandshakeSpec(
+            type = HelperFunction.lifecycleEvent.name,
+            version = 1
+        ),
+        HandshakeSpec(
+            type = HelperFunction.closeForm.name,
+            version = 1
+        )
+    )
```

Comment:
Moved handshake collection here for Native -> JS messages, makes more sense, similar to NativeBridge: https://github.com/klaviyo/klaviyo-android-sdk/blob/5873d71310c00a5a9b429d7cda58fedcc3c8127b/sdk/forms/src/main/java/com/klaviyo/forms/bridge/NativeBridge.kt#L25